kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn-pmml-trusty/pom.xml

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there any reason for this new module? There are already tests for DMN with PMML in kie-server-integ-tests-dmn-pmml. It might be easier to use the same tests for testing all three implementations and only switch the implementation using some profiles, params and assume etc. It will give us some confidence that all implementations can be triggered the same way in exec server.

Reply:
I agree it could be done, but working on such tests I see the following issues:
1) all testing projects (for all modules) uses the same GAV for their parent 
`  <parent>
    <groupId>org.kie.server.testing</groupId>
    <artifactId>common-parent</artifactId>
    <version>1.0.0.Final</version>
  </parent>`
 even if such parent is defined (= cloned) in all testing modules; that means that all those "common" parent must be identical and can contains only declarations common to *all* models (which I think it is somehow a nonsense)
2) inside  `kie-server-integ-tests-pmml` (see last commit) I defined a common parent GAV specific for those tests, and that allowed me to centralize all duplicated configurations in one single place (that is hugely better to read/debug/maintain)
3) I could not do the same if both "legacy" and "trusty" dmn-pmml tests would be in the same module
4) beside that, there are already profiles used to define the container (local/tomcat/wildfly, etc) and adding another one to define the pmml type (trusty vs legacy) would create a matrix hard to test/debug/maintain
5) last note, legacy is going to be completely removed ASAP, so this "duplication" would be in place only for the meantime (as all the other duplications/workarounds made specifically for the current transition time)
6) very last note: PMML rest invocation is always made through `ApplyPmmlModelCommand`; see [ApplyPmmlModelCommand#execute(Context context)](https://github.com/gitgabrio/drools/blob/9630c090db4dd3767942e5b21fb73563f6c21507/drools-core/src/main/java/org/drools/core/command/runtime/pmml/ApplyPmmlModelCommand.java#L136)

Please feel free to ping me if you would like to discuss more about that :smile: